Makefile

```diff
@@ -1,5 +1,5 @@
 
-ALL_DRIVERS := nifake nidcpower nidmm nifgen niscope niswitch nise nimodinst
+ALL_DRIVERS := nifake nidigital nidcpower nidmm nifgen niscope niswitch nise nimodinst
```

Comment:
The order here is random.
I had sort of reordered alphabetically when I did merges but that has the hidden behavior of changing the overall version with whatever is last on the list... just an observation.

A comment, even attached to a new issue about "last one wins" could be a good idea but feel free to drop.